en/manual/platforms/linux/setup-and-requirements.md

```diff
@@ -22,26 +22,56 @@ You need the following packages:
 
 To render fonts, we use the [FreeType](https://www.freetype.org/) library. The minimum required version is 2.6 and can be installed via:
 
-```
+# [Ubuntu](#tab/freetype-ubuntu)
+
+```bash
 sudo apt-get install libfreetype6-dev
 ```
 
+# [Fedora](#tab/freetype-fedora)
+
+```bash
+sudo dnf install freetype-devel
+```
+
+---
+
 ## OpenAL
 
 To play sounds and music, we use the [OpenAL](https://www.openal.org/) library. It can be installed via:
 
-```
+# [Ubuntu](#tab/openal-ubuntu)
```

Comment:
If Ubuntu and Fedora are sub-sections should they be a level down, meaning using title `###`?

Reply:
Good question, I mainly based it on the docfx example and as you can see it doesnt really matter : https://raw.githubusercontent.com/dotnet/docfx/main/docs/docs/markdown.md

Reply:
Ohh it does matter on the website 🙂, including it makes it harder for someone using a screen reader to understand the content if headings aren't used in the proper order.

- h1 = #
- h2 = ##
- h3 = ###
- h4 = ####
- and so on

You could actually see it directly in your repo that it should be a `###`

https://github.com/stride3d/stride-docs/blob/092266a68d2e84aebb5abe19079d87ed4822f92a/en/manual/platforms/linux/setup-and-requirements.md

![image](https://github.com/stride3d/stride-docs/assets/4528464/3a91f47c-a027-4b0f-889d-04a9923bda94)

Reply:
Right, I forgot about the "plain" markdown :) Is it better now? Btw, Shouldn't the .NET Core section be one level higher as a "separate" section? (From level 3 to level 2)

Reply:
You are right. Devs can code a game engine but they have hockey 🏒 in their markdown 😂. So even though I'm guiding you, the previous content might not be accurate. If you don't mind, could you correct the .NET Core section to level 2?

And it looks better, you can click **Display the rich diff**, to see the difference.

![image](https://github.com/stride3d/stride-docs/assets/4528464/6872eebb-11fd-4dc2-8afa-9aefe4e0e9ac)

Reply:
Cool, This should be final change and we're ready to go 🙏